src/internet_identity/internet_identity.did

```diff
@@ -158,13 +175,15 @@ type InternetIdentityInit = record {
         nat64;
         nat64;
     };
-    // Set the module hash of the archive canister
-    archive_module_hash : opt blob;
+    // Archive config parameters to use for the next archive deployment.
+    archive_config: opt ArchiveConfig;
```

Comment:
what does "next archive deployment" mean?

Also, if `archive_config: null` then no archive is deployed right? (just double checking)

Reply:
This should explain that the configuration is only (fully) applied if the archive is redeployed. I.e. changing the polling interval on the archive side requires an archive deployment.
I will add a clarification.

No, changing the II config does _not_ trigger an archive deployment, regardless of whether `archive_config` is set or not.